ocs_ci/ocs/constants.py

```diff
@@ -1095,6 +1095,9 @@
 
 PRODUCTION_JOBS_PREFIX = ["jnk"]
 
+# Cloud Manager available platforms
+cld_mgr_platforms = ["AWS", "GCP", "AZURE", "IBMCOS"]
```

Comment:
Sorry, what I meant was that this variable should be changed to `CLD_MGR_PLATFORMS`. All of our constants are in all caps, this one should be too.

Reply:
Ooohhh dear. Not my brightest moment. _At all_.
Sorry, and thank you :)